Tests/Unit/InMemoryAuthCodeManagerTest.php

```diff
@@ -20,53 +20,128 @@ public function testClearExpired(): void
+        $this->assertManagerContainsExpectedData($testData['output'], $inMemoryAuthCodeManager);
     }
 
-    private function buildClearExpiredTestData(): array
+    private function buildTestData(callable $successFunction): array
     {
-        $validAuthCodes = [
-            '1111' => $this->buildAuthCode('1111', '+1 day'),
-            '2222' => $this->buildAuthCode('2222', '+1 hour'),
-            '3333' => $this->buildAuthCode('3333', '+1 second'),
-            '4444' => $this->buildAuthCode('4444', 'now'),
+        $data = [
+            [
+                'identifier' => '1111',
+                'dateOffset' => '+1 day',
+                'revoked' => true,
+                'expired' => false,
+            ],
+            [
+                'identifier' => '2222',
+                'dateOffset' => '+1 hour',
+                'revoked' => false,
+                'expired' => false,
+            ],
+            [
+                'identifier' => '3333',
+                'dateOffset' => '+1 second',
+                'revoked' => true,
+                'expired' => false,
+            ],
+            [
+                'identifier' => '4444',
+                'dateOffset' => 'now',
+                'revoked' => false,
+                'expired' => false,
+            ],
+            [
+                'identifier' => '5555',
+                'dateOffset' => '-1 day',
+                'revoked' => true,
+                'expired' => true,
+            ],
+            [
+                'identifier' => '6666',
+                'dateOffset' => '-1 hour',
+                'revoked' => false,
+                'expired' => true,
+            ],
+            [
+                'identifier' => '7777',
+                'dateOffset' => '-1 second',
+                'revoked' => true,
+                'expired' => true,
+            ],
         ];
 
-        $expiredAuthCodes = [
-            '5555' => $this->buildAuthCode('5555', '-1 day'),
-            '6666' => $this->buildAuthCode('6666', '-1 hour'),
-            '7777' => $this->buildAuthCode('7777', '-1 second'),
-        ];
+        $response = [];
+        foreach ($data as $item) {
+            $identifier = $item['identifier'];
+            $authCode = $this->buildAuthCode(
+                $identifier,
+                $item['dateOffset'],
+                $item['revoked']
+            );
+            $response['input'][$identifier] = $authCode;
 
-        return [
-            'input' => $validAuthCodes + $expiredAuthCodes,
-            'output' => $validAuthCodes,
-        ];
+            if ($successFunction($item)) {
+                $response['output'][$identifier] = $authCode;
+            }
+        }
+
+        return $response;
     }
 
-    private function buildAuthCode(string $identifier, string $modify): AuthorizationCode
+    private function buildAuthCode(string $identifier, string $modify, bool $revoked): AuthorizationCode
```

Comment:
@franjo-zadelj-trikoder Another inconsistency. In some cases the $revoked argument has a default value `false`, here it doesn't. Please make that consistent.

Reply:
explained above